### PR TITLE
SimG4CMS/Forward : gcc 6.0 misleading-indentation warning flags potential bug; with bug fix

### PR DESCRIPTION
**Pull request by gartung**
/build/cmsbuild/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/8258e37281729e179b5af9c3b033d930/opt/cmssw/slc6_amd64_gcc600/cms/cmssw/CMSSW_8_1_X_2016-06-26-2300/src/SimG4CMS/Forward/src/CastorTestAnalysis.cc:44:3: warning: this 'if' clause does not guard... [-Wmisleading-indentation]
    if (verbosity > 0)
   ^~
/build/cmsbuild/jenkins-workarea/workspace/build-any-ib/w/tmp/BUILDROOT/8258e37281729e179b5af9c3b033d930/opt/cmssw/slc6_amd64_gcc600/cms/cmssw/CMSSW_8_1_X_2016-06-26-2300/src/SimG4CMS/Forward/src/CastorTestAnalysis.cc:46:4: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the 'if'
    std::cout<<"============================================================================"<<std::endl;
    ^~~

### CODE REVIEW

**Comment by civanch** [comment]
please test

File: SimG4CMS/Forward/src/CastorTestAnalysis.cc [hunk 1]
```diff
@@ -41,20 +41,19 @@ CastorTestAnalysis::CastorTestAnalysis(const edm::ParameterSet &p) {
   stepNtFileName           = m_Anal.getParameter<std::string>("StepNtupleFileName");
   eventNtFileName          = m_Anal.getParameter<std::string>("EventNtupleFileName");
 
-  if (verbosity > 0)
-   std::cout<<std::endl;
-   std::cout<<"============================================================================"<<std::endl;
```
Comment (gartung) [inline SimG4CMS/Forward/src/CastorTestAnalysis.cc]: Is this statement part of the if block?

**Comment by civanch** [comment]
please test

File: SimG4CMS/Forward/src/CastorTestAnalysis.cc [hunk 2]
```diff
@@ -41,20 +41,19 @@ CastorTestAnalysis::CastorTestAnalysis(const edm::ParameterSet &p) {
...
-   if (doNTcastorstep  > 0){
-     std::cout <<" Step Ntuple will be created"<< std::endl;
-     std::cout <<" Step Ntuple file: "<<stepNtFileName<<std::endl;
-   }
-   if (doNTcastorevent > 0){
-     std::cout <<" Event Ntuple will be created"<< std::endl;
-     std::cout <<" Step Ntuple file: "<<stepNtFileName<<std::endl;
-   }
-   std::cout<<"============================================================================"<<std::endl;
-   std::cout<<std::endl;
```
Comment (gartung) [inline SimG4CMS/Forward/src/CastorTestAnalysis.cc]: Actually it looks like everything up to here is meant to be in the if statement from line 44.

**Comment by civanch** [comment]
please test
I agree - you are right

**Comment by civanch** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**